internal/backend/db/db.go

```diff
@@ -111,7 +111,7 @@ type DB interface {
 	DeleteConnection(ctx context.Context, id sdktypes.ConnectionID) error
 	GetConnection(ctx context.Context, id sdktypes.ConnectionID) (sdktypes.Connection, error)
 	GetConnections(ctx context.Context, ids []sdktypes.ConnectionID) ([]sdktypes.Connection, error)
-	ListConnections(ctx context.Context, filter sdkservices.ListConnectionsFilter, idsOnly bool) ([]sdktypes.Connection, error)
+	ListConnections(ctx context.Context, opts sdkservices.ListConnectionsFilter, idsOnly bool) ([]sdktypes.Connection, error)
```

Comment:
it's called filter all over the place, why opts ?

Reply:
ick leftover. fixed.